### DISCUSSION TITLE
Welcome to tourism-website Discussions!

### DISCUSSION THREAD

**Discussion by PritamSarbajna** [PritamSarbajna/tourism-website#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GPuneethReddy04** [comment]
why you did not  used any storage process like mongodb for the storage of the user details

**Comment by PritamSarbajna** [comment]
@GPuneethReddy04 Hey, which users are you talking about ??
